app/assets/javascripts/app/app_router.coffee

```diff
@@ -0,0 +1,21 @@
+Router = Backbone.Router.extend {
+  routes:
+    ':name'   : 'show'
+
+  show: (name) ->
+    @showPage(name)
+
+  showPage: (name) ->
+    window.peopleList = new PeopleList
+    peopleList.fetch().then ->
+      $('.card').remove()
+      $.each(peopleList._byId, (_prop, person_data) ->
```

Comment:
Backbone extends Model and Collection with a bunch of underscore methods built-in.  http://backbonejs.org/#Collection-Underscore-Methods

So to loop through a collection you could just do `peopleList.each (person) -> ...`